.github/workflows/lint.yml

```diff
@@ -0,0 +1,22 @@
+name: Lint ROS2 package
+# Run this action whenever a pull request is made
+on:
+  pull_request:
+
+# Specify the linting job IDs to process. 
```

Comment:
What does this mean?

Reply:
Specifying the job ID workflows: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_id

Not sure if there's a better way to put it.

Reply:
I see what you mean, actually ament_lint_cpp is the job id here, not "jobs". Maybe we can just drop this comment, I'm not sure how it helps user?